build/release.sh

```diff
@@ -6,6 +6,19 @@ export NODE_PATH=$NODE_PATH:./node_modules
 OLD_VERSION="XXX"
 VERSION="XXX"
 
+dry_run=false
```

Comment:
It looks like our current variables follow SNAKE_CAPS convention.

Reply:
Most of the variables do, our switches do not. Since this was a command line switch I made it look like the others